lottie/src/main/java/com/airbnb/lottie/network/Result.java

```diff
@@ -0,0 +1,31 @@
+package com.airbnb.lottie.network;
+
+import java.io.InputStream;
+
+import androidx.annotation.NonNull;
+
+public class Result {
```

Comment:
Let's rename this. `Result` will have lots of naming conflicts with other libraries that will be annoying when trying to use autocomplete. Can you reuse `LottieResult` or give this class a name that's specific to it being a network result?

Reply:
done